README.md

```diff
@@ -1,4 +1,10 @@
-## Purpose
+# [EOL] Node API
+
+**This repo is no longer updated.** Instead, use the `RuntimeClass` types
```

Comment:
Should we include specific version information? Something like: "The node-api group was migrated to a built-in api in the k8s.io/api with the v1.14 release. This repo is no longer maintained, and no longer synced with core kubernetes as of the v1.18 release."

Reply:
Done! :+1: